rotkehlchen/chain/ethereum/airdrops.py

```diff
@@ -74,17 +100,34 @@ def check_airdrops(
     found_data: Dict[ChecksumEthAddress, Dict] = defaultdict(lambda: defaultdict(dict))
     for protocol_name, airdrop_data in AIRDROPS.items():
         data, csvfile = get_airdrop_data(protocol_name, data_dir)
-        for addr, amount, *_ in data:
-            # not doing to_checksum_address() here since the file addresses are checksummed
-            # and doing to_checksum_address() so many times hits performance
-            if protocol_name in ('cornichon', 'tornado', 'grain'):
-                amount = token_normalized_value_decimals(int(amount), 18)
-            if addr in addresses:
-                found_data[addr][protocol_name] = {
-                    'amount': str(amount),
-                    'asset': airdrop_data[1],
-                    'link': airdrop_data[2],
-                }
+        timestamp_now = Timestamp(int(datetime.datetime.utcnow().timestamp()))
+        if protocol_name == 'furucombo':
+            if timestamp_now < FURUCOMBO_AIRDROP_ENDS_AT:  # do not display after week 9 finishes
+                for row in data:
+                    addr = row[0]
+                    if addr in addresses:
+                        amount = get_furucombo_claimable_combo(
+                            row=row,
+                            timestamp_now=timestamp_now,
+                        )
+                        found_data[addr][protocol_name] = {
```

Comment:
Vesting schemas are tricky and definitely we don't want to be very specific (front-end work). But we could put in "amount" the "total amount" (e.g. 350, 900 COMBO, we have it and generally all airdrops include it) and then add an extra "info" (or "details") text attribute where we can put the amount in context and advise the user. For instance: "This address is eligible to claim 350 COMBO that will be vested over eight weeks. Please, check the Furucombo airdrop details.".

Reply:
we will just put total for now as we do for all. This is not a balance return, but how much they can get

Reply:
> COMBO airdrop starts on the 2021-01-15 at 03:00:00 UTC with an amount that comes from the Week 0 header. Every Friday at 3:00 (+7 natural days) a new amount is vested (from Week 1, Week 2... headers). We return this amount during the whole week until next vesting point. After 9 natural weeks (a week past after the last vesting point, from Week 8 header) we do not return anymore COMBO amounts.

Why nothing after week 9? Does it stop being claimable? I don't see anything like that in the medium post.